client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MigrationClientDocumentationIT.java

```diff
@@ -80,4 +92,72 @@ public void testGetAssistance() throws IOException {
         }
         // end::get-assistance-response
     }
+
+    public void testUpgrade() throws IOException {
```

Comment:
here I am adding these test cases which are duplicates of the one I have in MigrationIT. I made more documentation friendly here though.

Reply:
++